content/en/dashboards/widgets/wildcard.md

```diff
@@ -41,6 +42,22 @@ However, if none of the Datadog widgets meets your visualization needs, a Wildca
 1. (Optional) Debug Vega-Lite specification mismatches with [Data Preview](#data-preview). Make sure the query in your Vega-Lite specification maps to the Datadog query. 
 1. Click **Save**.
 
+#### Formulas Scalar vs. Formulas Timeseries
+
+In Datadog dashboards, scalar and timeseries visualizations serve different purposes and are used to represent data in distinct ways.
+
+**Timeseries**
+: This visualization is designed to display how your data changes over time.
+   - **Use-cases**: It's ideal for monitoring metrics that fluctuate, such as CPU usage, memory consumption, or request rates. It helps identify trends, patterns, and anomalies over a specified time range. 
+   - **Appearance**: These visualizations are typically line or area charts. They show datapoints plotted against time (usually on the x-axis), to observe changes and trends.
+
+**Scalar**
+: This visualization presents a single, aggregate value derived from your data.
```

Comment:
This describes the "query value" case, but doesn't fully describe how the scalar endpoint is used. The scalar endpoint powers the toplist, treemap, sumburst, and table widgets because 1 value is produced per `groupBy` and per `formula` .

I think that referring to these two sections as describing "data formats" rather than "visualization types" may make this easier to follow.

I think the takeaway that "if you need to describe a change over time, use timeseries, otherwise prefer scalar" works well for this section. 

We may also want to touch briefly on why "Formulas" appears in the title. The other two request formats (histogram, list), are not compatible with [Functions](https://docs.datadoghq.com/dashboards/functions/) API (also called "Formulas and Functions" / "F+F" internally), which is why only Scalar/Timeseries have the formulas prefix.